crates/re_query_cache/src/cache_stats.rs

```diff
@@ -20,6 +20,19 @@ pub fn set_detailed_stats(b: bool) {
     ENABLE_DETAILED_STATS.store(b, std::sync::atomic::Ordering::Relaxed);
 }
 
+/// If `true`, will show stats about empty caches too, which likely indicates a bug (dangling bucket).
```

Comment:
meaning empty caches should be removed automatically?

Reply:
I find it a bit weird to encounter this atomic here. Isn't that just a property of the ui? We should put it on the `MemoryPanel` struct instead.
Personally, I'd just always enable this anyways doesn't seem to hurt 🤷

Reply:
> meaning empty caches should be removed automatically?

Yes, or not exist to begin with: either way, empty buckets are bug and we want to be able to see it.

> I find it a bit weird to encounter this atomic here. Isn't that just a property of the ui? We should put it on the `MemoryPanel` struct instead.

I just blindly followed how the other memory panel checkboxes are implemented :man_shrugging:.
I'll see if I can easily make it 

> Personally, I'd just always enable this anyways doesn't seem to hurt 🤷 

No it's insanely costly with a real dataset

Reply:
thanks for upgrading all of them, liking this much better :)